instance-configs/Config.cprobe.groovy

```diff
@@ -0,0 +1,716 @@
+ * the transmart-data checkout. That file will be appended to this one whenever
+ * the Config.groovy target is run */
+
+environments { production {
+    if (transmartURL.startsWith('http://localhost:')) {
+        println "[WARN] transmartURL not overridden. Some settings (e.g. help page) may be wrong"
+    }
+} }
+
+/* {{{ Log4J Configuration */
+log4j = {
+    environments {
+        development {
+            root {
+                info 'stdout'
+            }
+
+            // for a less verbose startup & shutdown
+            warn  'org.codehaus.groovy.grails.commons.spring'
+            warn  'org.codehaus.groovy.grails.orm.hibernate.cfg'
+            warn  'org.codehaus.groovy.grails.domain.GrailsDomainClassCleaner'
+
+            debug 'org.transmartproject'
+            debug 'com.recomdata'
+            debug 'grails.app.services.com.recomdata'
+            debug 'grails.app.services.org.transmartproject'
+            debug 'grails.app.controllers.com.recomdata'
+            debug 'grails.app.controllers.org.transmartproject'
+            debug 'grails.app.domain.com.recomdata'
+            debug 'grails.app.domain.org.transmartproject'
+            // debug 'org.springframework.security'
+            // (very verbose) debug  'org.grails.plugin.resource'
+        }
+
+        production {
+            def logDirectory = "${catalinaBase}/logs".toString()
+            appenders {
+                rollingFile(name: 'transmart',
+                            file: "${logDirectory}/transmart.log",
+                            layout: pattern(conversionPattern: '%d{dd-MM-yyyy HH:mm:ss,SSS} %5p %c{1} - %m%n'),
+                            maxFileSize: '100MB')
+            }
+            root {
+                warn 'transmart'
+            }
+        }
+    }
+}
+/* }}} */
+
+/* {{{ Faceted Search Configuration */
+environments {
+    development {
+        com.rwg.solr.scheme = 'http'
+        com.rwg.solr.host   = 'localhost:8983'
+        com.rwg.solr.path   = '/solr/rwg/select/'
+    }
+
+    production {
+        com.rwg.solr.scheme = 'http'
+        com.rwg.solr.host   = 'localhost:' + solrPort
+        com.rwg.solr.path   = '/solr/rwg/select/'
+    }
+}
+/* }}} */
+
+/* {{{ Data Upload Configuration - see GWAS plugin Data Upload page */
+// This is the value that will appear in the To: entry of the e-mail popup 
+// that is displayed when the user clicks the Email administrator button,
+// on the GWAS plugin Data Upload page
+com.recomdata.dataUpload.adminEmail = 'No data upload adminEmail value set - contact site administrator'
+/* }}} */
+
+/* {{{ Personalization */
+// application logo to be used in the login page
+com.recomdata.largeLogo = "cprobeLogo.png"
+
+// application logo to be used in the search page
+com.recomdata.searchtool.smallLogo="cprobeLogo.png"
+
+// contact email address
+com.recomdata.contactUs = "mailto:NeptuneTMSupport@umich.edu"
```

Comment:
Guessing we should change both of the contactUs addresses here.

Reply:
This is the same as production. Let's check with Becky.